test/test.py

```diff
@@ -71,18 +71,32 @@ def test_score_combined(self):
         # Compare with a reference.
         self.assertTrue(_logs_equal(_LOG, _REF_UPDATED))
 
-if __name__ == '__main__':
-    print('Integration tests: ', end='')
-
-    # Log file not found test.
-    fs_py = ['python', os.path.join('src', 'facere_sensum', 'facere_sensum.py'),
-             '--log', 'llog.csv', 'update']
-    res = subprocess.run(fs_py, check=False, capture_output=True, text=True).stdout # nosec B603
-    if res == 'Log file \'llog.csv\' not found. Exiting.\n':
+def _test_integration(descr, args, ref):
+    '''
+    Run an integration test.
+    'descr' is test user description.
+    'args' command line arguments to use.
+    'ref' expected output.
+    '''
+    print(descr, end=': ')
+    res = subprocess.run(['python',
+                         os.path.join('src', 'facere_sensum', 'facere_sensum.py')] + args,
+                         check=False, capture_output=True, text=True).stdout # nosec B603
+    if res == ref:
         print('OK')
     else:
-        print('FAILED. Output:', res)
+        print('FAILED', 'Output:', '---', res, '---', 'Expected:', '---', ref, '---', sep='\n')
         sys.exit(1)
 
+if __name__ == '__main__':
+    print('Integration tests:')
+    _test_integration('Authentication config not found',
+                      ['--auth', 'aauth.json', 'update'],
```

Comment:
`aauth.json` or `auth.json`?

Reply:
'aauth.json' is intentional since the test is checking for what happens if authentication config file is not found. Maybe I can use 'notfound.json' to make this explicit :) Will that be better?

Reply:
yes

Reply:
Fixed in branch